modules/core/src/main/scala/org/scalasteward/core/update/PruningAlg.scala

```diff
@@ -37,20 +37,32 @@ final class PruningAlg[F[_]](
     implicit
     dateTimeAlg: DateTimeAlg[F],
     logger: Logger[F],
+    repoConfigAlg: RepoConfigAlg[F],
```

Comment:
just realized RepoConfig is already part of RepoCache, so injecting RepoConfigAlg doesn't  make sense here